lib/auth/webauthntypes/webauthn.go

```diff
@@ -376,3 +379,50 @@ type Credential protocol.Credential
 type AuthenticationExtensionsClientOutputs struct {
 	AppID bool `json:"appid,omitempty"`
 }
+
+// SessionData is a clone of [webauthn.SessionData], materialized here to keep a
+// stable JSON marshal/unmarshal representation and add extensions.
```

Comment:
This seems to have diverged from the version of the `webauthn.SessionData` we're using (it's missing `Expires` and `Extensions`)

https://pkg.go.dev/github.com/go-webauthn/webauthn@v0.10.0/webauthn#SessionData

(Unclear to me whether that's a bug in the code or the comment)

Reply:
There's a TODO from @codingllama to add extensions. I'll update and move it to mention expires as well. IIUC we are not using either of these fields currently so we don't have a reason to record them as of now.

Reply:
It's alright, we don't need every feature from upstream. Other messages probably have small divergences too.